easybuild/easyconfigs/t/tdlib/tdlib-0.9.3-GCC-13.2.0.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'ConfigureMake'
+
+name = 'tdlib'
+version = '0.9.3'
+
+homepage = 'https://github.com/freetdi/tdlib/'
+description = """treedec provides tree decomposition algorithms."""
+
+toolchain = {'name': 'GCC', 'version': '13.2.0'}
+
+source_urls = ['https://github.com/freetdi/tdlib/archive/']
+sources = ['%(version)s.tar.gz']
+checksums = ['d1730c98f41dcb23bbd0bd8de9dbec51df015304f28a38935848925901594ae8']
+
+builddependencies = [
+    ('Autotools', '20220317'),
+    ('Boost', '1.83.0'),
+]
+
+dependencies = [
+    ('Python', '3.11.5'),
```

Comment:
@PetrKralCZ How is Python a runtime dependency here exactly?

Reply:
This should only be a build dependency. Fixed in [31ce7fc](https://github.com/easybuilders/easybuild-easyconfigs/pull/21305/commits/31ce7fc97a82a6d1960d684fb0052e7fe5cdb218).